windows_containers/enabling-windows-container-workloads.adoc

```diff
@@ -28,7 +28,7 @@ Dual NIC is not supported on WMCO-managed Windows instances.
 
 [NOTE]
 ====
-The WMCO is not supported in clusters that use a xref:../networking/enable-cluster-wide-proxy.adoc#enable-cluster-wide-proxy[cluster-wide proxy] because the WMCO is not able to route traffic through the proxy connection for the workloads.
+Windows instances deployed by the WMCO are configured with the containerd container runtime. Because WMCO installs and manages the runtime, it is recommanded to not manually install containerd in nodes.
 ====
```

Comment:
Added new note; moved existing note to [following section](https://github.com/openshift/openshift-docs/pull/47588/files#diff-264a4dd7625b8e6d20906acb2c76d430c917c0f23caba9e6eb923919611c0492R43) to avoid stacked notes.